spec/unit/provider/user/smartos_spec.rb

```diff
@@ -0,0 +1,91 @@
+#
+# Author:: Dave Eddy (<dave@daveeddy.com>)
+# License:: Apache License, Version 2.0
```

Comment:
Technically you should have a copyright line in here, that goes along with the license. (You can only license something that is copyrighted). For example:
```
# Copyright:: Copyright 2015, Dave Eddy
```

Or whatever, like if you're doing this work for a company and they retain your copyright, like we put Chef Software, Inc. in there.